components/sup/src/manager/mod.rs

```diff
@@ -87,7 +87,10 @@ impl Manager {
                                                  None));
         outputln!("Butterfly Member ID {}", server.member_id());
         for peer_addr in gconfig().gossip_peer() {
-            let addr: SocketAddr = peer_addr.parse().unwrap();
+            let addrs: Vec<SocketAddr> = peer_addr.to_socket_addrs()
+                                                  .expect("Unable to resolve domain")
```

Comment:
This function returns a `Result` - we should add an error for failure to resolve, and return an error, so that the tooling can handle things like pretty printing the error.